### PR TITLE
Rename .tfvars files to add .example extension

### PR DESCRIPTION
**Pull request by arbulu89**
Rename .tfvars to add .example extension to avoid issues uploading sensitive information.

(some files are formatted by my editor removing some not needed spaces)

### CODE REVIEW

File: aws/terraform/README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ These are the relevant files and what each provides:
 
 - [remote-state.tf](remote-state.tf): definition of the backend to [store the Terraform state file remotely](create_remote_state).
 
-- [terraform.tfvars](terraform.tfvars): file containing initialization values for variables used thoughout the configuration. **Edit this file with your values before use**.
+- [terraform.tfvars.example](terraform.tfvars.example): file containing initialization values for variables used thoughout the configuration. **Rename/Duplicate this file to terraform.tfvars and edit the content with your values before use**.
```
Comment (alvarocarvajald) [inline aws/terraform/README.md]: Missing an 'r' in throughout (which it seems was there before and we didn't catch it before).

File: aws/terraform_salt/README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ These are the relevant files and what each provides:
 
 - [remote-state.sample](remote-state.sample): sample file for the definition of the backend to [store the Terraform state file remotely](create_remote_state).
 
-- [terraform.tfvars](terraform.tfvars): file containing initialization values for variables used throughout the configuration. **Edit this file with your values before use**.
+- [terraform.tfvars.example](terraform.tfvars.example): file containing initialization values for variables used thoughout the configuration. **Rename/Duplicate this file to terraform.tfvars and edit the content with your values before use**.
```
Comment (alvarocarvajald) [inline aws/terraform_salt/README.md]: Same issue with throughout.

File: azure/terraform/README.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ These are the relevant files and what each provides:
 
 * [outputs.tf](outputs.tf): definition of outputs of the terraform configuration.
 
-* [terraform.tfvars](terraform.tfvars): file containing initialization values for variables used thoughout the configuration. **Edit this file with your values before use**.
+* [terraform.tfvars.example](terraform.tfvars.example): file containing initialization values for variables used thoughout the configuration. **Rename/Duplicate this file to terraform.tfvars and edit the content with your values before use**.
```
Comment (alvarocarvajald) [inline azure/terraform/README.md]: Same issue with throughout.

File: gcp/terraform/README.md [hunk 4]
```diff
@@ -73,13 +73,13 @@ These are the relevant files and what each provides:
 
  - [templates.tf](templates.tf): definition of the templates used.
 
- - [variables.tf](variables.tf): definition of variables used in the configuration. 
- 
- - [terraform.tfvars](terraform.tfvars): defaults for the variables defined in [variables](variables.tf)
+ - [variables.tf](variables.tf): definition of variables used in the configuration.
+
+ - [terraform.tfvars.example](terraform.tfvars.example): file containing initialization values for variables used thoughout the configuration. **Rename/Duplicate this file to terraform.tfvars and edit the content with your values before use**.
```
Comment (alvarocarvajald) [inline gcp/terraform/README.md]: Same issue with throughout.

File: gcp/terraform/README.md [hunk 5]
```diff
@@ -73,13 +73,13 @@ These are the relevant files and what each provides:
...
 
 ## How to use
 
-1. Edit [terraform.tfvars](terraform.tfvars) to suit your needs or use `terraform [-var VARIABLE=VALUE]...` to override defaults.
+1. Rename [terraform.tfvars.example](terraform.tfvars.example) to *terraform.tfvars* and edit suit your needs or use `terraform [-var VARIABLE=VALUE]...` to override defaults.
```
Comment (alvarocarvajald) [inline gcp/terraform/README.md]: Missing "to" between "edit" and "suit"

**Comment by arbulu89** [comment]
Thanks @alvarocarvajald I didn't notice these typos. All of them fixed!
Let me know if you find more typos, I have not done a extensive check

**Comment by alvarocarvajald** [comment]
LGTM

**Pull request merged**